### PR TITLE
Add image as notification attachment

### LINKED ISSUE #236: LocalNotifications on iOS don't show image
Notifications can show an image, but this only works on Android, on iOS it has to be implemented as an Attachment.

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #236

### CODE REVIEW

File: modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java [hunk 1]
```diff
@@ -52,10 +72,42 @@ protected void unscheduleNotification(Notification notification) {
...
+    private void createImageInAssets(InputStream imageInputStream) {
+        File file = new File(assetsFolder, NOTIFICATION_IMAGE);
+        if (file.exists()) {
+            file.delete();
+        }
+
+        try (imageInputStream) {
+            copyFile(imageInputStream, file.getAbsolutePath());
+        } catch (IOException ex) {
+            LOGGER.log(Level.WARNING, "Error reading image file", ex);
```
Comment (johanvos) [inline modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java:89]: Shouldn't the caller somehow be informed that copying failed?

File: modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java [hunk 2]
```diff
@@ -52,10 +72,42 @@ protected void unscheduleNotification(Notification notification) {
...
+
+        try (OutputStream os = new FileOutputStream(pathEnd)) {
+            byte[] buffer = new byte[1024];
+            int length;
+            while ((length = is.read(buffer)) > 0) {
+                os.write(buffer, 0, length);
+            }
+            os.flush();
+        } catch (IOException ex) {
+            LOGGER.log(Level.WARNING, "Error copying file", ex);
```
Comment (johanvos) [inline modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java:106]: IOException is catched here, but also in the caller (line 88). If an IO exception occurs during e.g. is.read(buffer), the caller won't know.

File: modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java (hunk 1 above)
Comment (jperedadnr) [reply modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java:89]: If the copying failed, there is already an exception logged. Probably not critical enough to interrupt the whole notification process?
Or do we need to foward the exception up to the API level?

File: modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java (hunk 2 above)
Comment (jperedadnr) [reply modules/local-notifications/src/main/java/com/gluonhq/attach/localnotifications/impl/IOSLocalNotificationsService.java:106]: Same as above

**Review by johanvos — APPROVED** [review]

**Pull request merged**